keith-vscode/package.json

```diff
@@ -160,6 +160,11 @@
       }
     ],
     "commands": [
+      {
+        "command": "keith-vscode.import-stpa-ltl",
+        "title": "Import LTL from STPA",
```

Comment:
here shouldn't it be PASTA? Since the import is coming from the PASTA extension

Reply:
but PASTA will support other languages as well and the imported LTL will come from STPA